virttest/qemu_vm.py

```diff
@@ -1676,7 +1673,9 @@ def sort_key(dev):
             add_virtio_rng(devices, rng_params, parent_bus)
 
         # Add logging
-        devices.insert(StrDev('isa-log', cmdline=add_log_seabios(devices)))
+        if (devices.has_device("isa-debugcon") and
```

Comment:
better to add   "enable_debug_console"  parameter in  test.cfg  like:
``enable_debug_console = yes

Reply:
do you mean put `enable_debug_console = yes`  in `avocado-vt/backends/qemu/cfg`

Reply:
yes

Reply:
updated, please have a look, thanks